app/config/routing.yml

```diff
@@ -7,6 +7,11 @@
 app:
     resource: @AppBundle/Controller/
     type:     annotation
+    prefix:   /{_locale}
+    requirements: 
+        _locale: en|fr|de|es|cs|ru|uk| 
```

Comment:
I think we should move accepted locales to `parameters.yml` too, for example:
``` yaml
# parameters.yml
parameters:
    locales: en|fr|de|es|cs|ru|uk|
```
So we could easily refers to it in other routes in future.
What do you think?